### DISCUSSION TITLE
LiveBot v1.2.0 alpha release - Twemoji & Barry are back from the dead

### DISCUSSION THREAD

**Discussion by SebOuellette** [SebOuellette/LiveBot#230] [General]
Huge thanks to this release's contributors! Barry is working properly again, member popout menus now show their role names, and the way emojis are handled has been modified.

I was kinda forced to make a release earlier than I would have liked to because of maxcdn changing names and not wanting to host twemoji anymore.

Anything before v1.2.0 is no longer functional unless the exact same maxcdn url comes back online. I also don't know who owns maxcdn, so it could come back online as something dangerous, not sure what's going to happen there. Anyway, use 1.2.0 or latest source commit to be up to date. (1.2.0 will still be up to date with all twemoji stuff without needing to update further)

## What's Changed
* Fix the nickname color by @Xaliks in https://github.com/SebOuellette/LiveBot/pull/205
* Fix typos by @Xaliks in https://github.com/SebOuellette/LiveBot/pull/206
* Fix the `titlebar` by @Xaliks in https://github.com/SebOuellette/LiveBot/pull/209
* Update & add new emojis and shortcuts by @Xaliks in https://github.com/SebOuellette/LiveBot/pull/207
* Fix Barry by @Xaliks in https://github.com/SebOuellette/LiveBot/pull/208
* Fix emojis by @Xaliks in https://github.com/SebOuellette/LiveBot/pull/210
* Add support for emojis by @JustMeCodes in https://github.com/SebOuellette/LiveBot/pull/224
* Made member roles more discord - like looking by @RedNight-was-taken in https://github.com/SebOuellette/LiveBot/pull/221

## New Contributors
* @Xaliks made their first contribution in https://github.com/SebOuellette/LiveBot/pull/205
* @JustMeCodes made their first contribution in https://github.com/SebOuellette/LiveBot/pull/224
* @RedNight-was-taken made their first contribution in https://github.com/SebOuellette/LiveBot/pull/221

**Full Changelog**: https://github.com/SebOuellette/LiveBot/compare/v1.1.5-alpha...v1.2.0-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/SebOuellette/LiveBot/releases/tag/v1.2.0-alpha'>LiveBot v1.2.0 alpha release - Twemoji & Barry are back from the dead</a>.</em>

**Comment by 3kh0** [comment]
yay

**Comment by Bon-Gaming** [comment]
Can you make a 32 bit version?
